FredLin.md

```diff
@@ -1,39 +1,61 @@
 [Open bugs assigned to me](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=assignee%3Agasolin%40mozilla.com) (ASSIGNED = current working on; NEW = backlog)
 
-## 8/8 ~ 8/12
+## 8/15 ~ 8/19
+
+Attend Coscup at 8/20~21.
+Present about webby project, Slide https://hackmd.io/s/HJ7dfgVc#/
```

Comment:
I assume this is the last time I will be hearing about Webby as your official work?